mfc.sh

```diff
@@ -107,6 +106,7 @@ if [ "$1" == 'load' ]; then
     # Find length of longest module_name in $MODULES for $COMPUTER
     max_module_length="0"
     for module_name in ${MODULES[@]}; do
+        echo "$module_name"
```

Comment:
Did you intend to include this change?

Reply:
@henryleberre, same here. There should be no (intended) changes to this file other than the ones I followed from you